taichi/transforms/make_block_local.cpp

```diff
@@ -211,23 +206,46 @@ void make_block_local_offload(OffloadedStmt *offload) {
         for (int i = 0; i < dim; i++) {
           // BLS index = sum_i inc_i
           // where inc_i =
-          //   bls_stride_i * (gbl_idx_i - loop_base_i - bls_lower_bound_i +
-          //   offset_i)
```

Comment:
Is this to temporarily disable the support of `for` index offsets?

Reply:
Actually here the `for` index offset is part fo `bls_lower_bound_i`. Let me update the comments for more clarity. Thanks for pointing this out!